src/managers/GCommandLoader.js

```diff
@@ -48,23 +49,24 @@ class GCommandLoader {
      * @private
      */
     async __loadCommandFiles() {
-        for await (let file of (await fs.readdirSync(this.cmdDir))) {
-            const fileTypeIndex = (file.lastIndexOf('.') - 1 >>> 0) + 2;
-            const fileName = file.slice(0, fileTypeIndex - 1);
-            const fileType = file.slice(fileTypeIndex);
+        for await (let fsDirent of (await fs.readdirSync(this.cmdDir, { withFileTypes: true }))) {
```

Comment:
I'm confused why we're using `for await...of` or `await` at all since [`fs.readdirSync`](https://nodejs.org/api/path.html#path_path_basename_path_ext) is not asynchronous.

Reply:
Not sure, my only addition to this line is `{ withFileTypes: true }`.

Reply:
Okay, we can turn this into a standard `for...of` loop using `const` seeing as `fsDirent` is never reassigned.